perun-base/src/main/java/cz/metacentrum/perun/rpc/deserializer/JsonDeserializer.java

```diff
@@ -64,9 +64,6 @@
 	@JsonIgnoreProperties({"shortName", "beanName"})
 	private interface GroupMixIn {}
 
-	@JsonIgnoreProperties({"attributes", "beanName"})
-	private interface ExtSourceMixIn {}
```

Comment:
Do not remove all ExtSourceMixIn, just attributes. We still want to ignore beanName!